packages/react-server/core/renderMiddleware.js

```diff
@@ -299,10 +299,12 @@ function writeHeader(req, res, context, start, pageObject) {
 		// PLAT-602: inline scripts come before stylesheets because
 		// stylesheet downloads block inline script execution.
 		renderScripts(pageObject, res),
-		renderStylesheets(pageObject, res),
-		renderMetaTags(pageObject, res),
-		renderLinkTags(pageObject, res),
-		renderBaseTag(pageObject, res),
+		renderStylesheets(pageObject, res)
+			.then(() => Q.all([
+				renderMetaTags(pageObject, res),
+				renderLinkTags(pageObject, res),
+				renderBaseTag(pageObject, res),
+			])),
```

Comment:
This forces meta/link/base writing _after_ stylesheets. I don't know if it's necessary, but I wanted to draw attention to this section and any ordering issues I might be introducing.